#! /bin/sh -e

# DP: Add hint to the doxygen-doc package in the manpages.

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

--- a/doc/doxygen.1	2005-01-17 20:52:35.000000000 +0100
+++ b/doc/doxygen.1	2004-09-21 09:39:28.216711000 +0200
@@ -42,3 +42,6 @@
 If configName is omitted `Doxyfile' will be used as a default.
 .SH SEE ALSO
 doxytag(1), doxywizard(1).
+.PP
+Please consider installing the doxygen-doc package as it contains additional
+documentation on how to use doxygen.
--- a/doc/doxywizard.1	2005-01-17 20:40:41.000000000 +0100
+++ b/doc/doxywizard.1	2004-09-21 09:38:16.164312000 +0200
@@ -8,3 +8,6 @@
 and run doxygen on your source files.
 .SH SEE ALSO
 doxygen(1), doxytag(1).
+.PP
+Please consider installing the doxygen-doc package as it contains additional
+documentation on how to use doxygen.
--- a/doc/doxytag.1	2005-01-17 20:40:50.000000000 +0100
+++ b/doc/doxytag.1	2004-10-09 08:01:49.156948000 +0200
@@ -18,3 +18,6 @@
 have a .html extension are parsed.
 .SH SEE ALSO
 doxygen(1), doxywizard(1).
+.PP
+Please consider installing the doxygen-doc package as it contains additional
+documentation on how to use doxygen.
